lib/validate-selectors.js

```diff
@@ -2,6 +2,56 @@ var listSequences = require('./list-sequences');
 var shouldIgnoreRule = require('./should-ignore-rule');
 var shouldIgnoreSelector = require('./should-ignore-selector');
 var toInterpoloatedRegexp = require('./to-interpolated-regexp');
+var resolveNestedSelector = require('postcss-resolve-nested-selector');
+
+function isTopLevel(node) {
+  return node.type === 'root';
+}
+
+function isNestedRule(node) {
+  return /(?:at)?rule/.test(node.parent.type)
+}
+
+function hasNoDeclarations(node) {
+  return node.nodes.length && node.every(function(child) {
```

Comment:
This is needed because if the array is empty then `node.every` just returns `true` which is incorrect, but if the array length was greater than `0` it still needs to be checked to ensure each item is a declaration.